backend/graph/api/collection-resolver.go

```diff
@@ -1,9 +1,9 @@
-package graph
+package graphapi
 
 import (
 	"context"
 	"github.com/bcc-code/brunstadtv/backend/common"
-	gqlmodel "github.com/bcc-code/brunstadtv/backend/graph/model"
+	gqlmodel2 "github.com/bcc-code/brunstadtv/backend/graph/api/model"
```

Comment:
Don't quite get why we have 2 here now?

Reply:
Me neither. gqlgen has it's reasons I guess...
Will do some cleanup

Reply:
Ah, didn't realize this was codegen. Don't care in that case :)

Reply:
Do you think the new package names will create any issues?

They should never be mixed, so as long as you only import what you need, I think it makes sense (also if we add more endpoints later we won't run out of package names)

Reply:
No, I don't think it's an issue in general, just reacted to the non-descriptive name, but for genned code it's not an issue.